packages/block-editor/src/store/test/actions.js

```diff
@@ -321,6 +321,7 @@ describe( 'actions', () => {
 				value: {
 					type: 'INSERT_BLOCKS',
 					blocks: [ block ],
+					patternName: false,
```

Comment:
if we did switch to undefined as the default we could add another test that checks it works correctly for when patternName is defined and when it isn't

Reply:
Added the test. :+1: